README.adoc

```diff
@@ -15,7 +15,7 @@ image::docs/images/Ui2.png[width="600"]
 
 endif::[]
 
-* Have you ever lost track of your expenditure and burst your budget for the month? Fret not! The Expense Tracker is here to help you.
+* Have you ever lost track of your expenditure and burst your totalBudget for the month? Fret not! The Expense Tracker is here to help you.
```

Comment:
I think this should still be 'budget' since this part is intended to be read by normal users.